### DISCUSSION TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'x')

### DISCUSSION THREAD

**Discussion by prashanta0234** [xyflow/xyflow#3515] [Q&A] [answered]
I copied the full code from https://reactflow.dev/docs/examples/edges/floating-edges/ and i am using version:11.6.1 , but nothing is shown in my browser. In the console I get an Uncaught TypeError: Cannot read properties of undefined (reading 'x') error. I`m using react-vite. How do I fix it?

**Comment by moklick** [comment, marked as answer]
could you first try to update to the latest version 11.9.4?

**Reply by prashanta0234** [reply]
Thank you
